### ISSUE TITLE
mpv can't find socket.url

### ISSUE THREAD

**Issue by Fuuzetsu** [NixOS/nixpkgs#1949]
mpv doesn't seem to be packaged properly or is missing a dependency. Trying to play any YouTube video using the regular `mpv YouTubelink` incantation, we get

```
** (process:5142): CRITICAL **: [_chk_script_ident] ...30/share/libquvi-scripts/0.9/common/quvi/youtube.lua:109: module 'socket.url' not found:
        no field package.preload['socket.url']
        no file './socket/url.lua'
        no file '/usr/share/lua/5.1/socket/url.lua'
        no file '/usr/share/lua/5.1/socket/url/init.lua'
        no file '/usr/lib/lua/5.1/socket/url.lua'
        no file '/usr/lib/lua/5.1/socket/url/init.lua'
        no file '/nix/store/6ihqlj1lyjnqaps626cpz4cq0iinvd6w-quvi-scripts-0.9.20131130/share/libquvi-scripts/0.9/common/socket/url.lua'
        no file './socket/url.so'
        no file '/usr/lib/lua/5.1/socket/url.so'
        no file '/usr/lib/lua/5.1/loadall.so'
        no file './socket.so'
        no file '/usr/lib/lua/5.1/socket.so'
        no file '/usr/lib/lua/5.1/loadall.so'
[quvi] Could not find any subtitle scripts in the path
[cache] Cache size set to 320 KiB
Cache fill: 20.62% (67584 bytes)   
Failed to recognize file format.


Exiting... (No files played)
```

I tried installing any related libs to no avail. Here's the relevant part of `nix-env -q`

```
lua-sockets-2.0.2
mpv-0.3.2
quvi-0.9.5
quvi-scripts-0.9.20131130
```

and here's `mpv --version`
```
mpv b'0.3.2' (C) 2000-2013 mpv/MPlayer/mplayer2 projects
 built on 2014-03-10T13:55:51
ffmpeg library versions:
   libavutil       52.48.101
   libavcodec      55.39.101
   libavformat     55.19.104
   libswscale      2.5.101
   libavfilter     3.90.100
   libavresample   1.1.0
```

Perhaps this needs to be refiled as libquvi issue but I leave the judgment to you.

**Comment by zimbatm** [comment]
I assume this have been fixed by the MPV 0.5.0 update. Please re-open if I'm wrong.

**Comment by AndersonTorres** [comment]
Yes, mpv now abandoned almost completely ytdl hack.
